### PR TITLE
Add comments to each of the metrics in telemeter to explain there use

### PR DESCRIPTION
**Pull request by smarterclayton**
Ensure we have clear documentation for why these metrics should be captured
and what their goal is

### CODE REVIEW

**Comment by squat** [comment]
Thanks Clayton. Can you fix the symlink at the root of the repo for metrics.json? (That file is there just to help surface the metrics and put them front and center)

File: jsonnet/telemeter/metrics.jsonnet [hunk 1]
```diff
@@ -0,0 +1,48 @@
+[
+    // up contains information relevant to the health of the registered
+    // cluster monitoring sources on a cluster. This metric allows telemetry
+    // identify when an update causes a service to begin to crashloop or flake.
+    "{__name__=\"up\"}",
```
Comment (squat) [inline jsonnet/telemeter/metrics.jsonnet]: Now that this is jsonnet and not strict json,I can make a follow up to clean up the quotes eg `'{__name__="up"}'`

File: jsonnet/telemeter/metrics.jsonnet [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[
+    // up contains information relevant to the health of the registered
+    // cluster monitoring sources on a cluster. This metric allows telemetry
+    // identify when an update causes a service to begin to crashloop or flake.
```
Comment (squat) [inline jsonnet/telemeter/metrics.jsonnet]: Nit: missing “to” between “telemetry” and “identify”

**Comment by smarterclayton** [comment]
Updated

**Pull request merged**